pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -38,6 +38,7 @@ var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-mgr")
 
 const (
 	appName     = "rook-ceph-mgr"
+	daemonName  = "a"
```

Comment:
Are we "100%" sure we never want to run more than one MGR?
I would prefer if we can go back to running two mgrs and with that revert the 'Revert "Run two MGRs to have one in standby mode"' #1334. :)

How about just converting the number to char https://stackoverflow.com/a/36804185 ?

Reply:
Note sure at all.  I think .a works just because there is only one and we need *some* suffix (=name).  If we run multiples, it'll be the gibberish that kubernetes uses to name the pod.

As long as the pod name matches the daemon name, and we don't have a bunch of redundant
"rook-ceph-mgr" pod name prefix goop in the daemon name (the thing after mgr. or mds. etc) then I'm happy!

Reply:
@galexrt I would propose we stick with one mgr for now since that is all that was already implemented anyway. We would want to create a standby with a separate deployment instead of using the replicaset count as previously. I'm thinking that when #1751 is implemented with the suffix calculated for `a`, we should also use the same calculation for the mgr here instead of hard coding to `a`. Then with the same calculation it would be simple to implement the standby mgr if needed.

Reply:
@travisn The previous code did not use `replicas: 2`, it spawned two separate Deployments, that is why I'm asking.
See https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/mgr/mgr.go#L84-L100 on line 91 it is creating a separate deployment for each "c.Replicas".

Reply:
@galexrt oh right, I was thinking of another place where it was hard-coded to a single deployment.